js/script.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function daysInYear(year) {
+  try {
+    if (!Number.isInteger(year) || !Number(year) > 0){
+      throw new Error();
+    }
+    console.log(daysOnLeapYear(year) ? 366 : 365);
+  } catch (error) {
+      console.log('exception');
```

Comment:
Based on requirements `it should throw an exception`. This line prevents the exception from being thrown.